app/src/test/java/protect/card_locker/contentprovider/AboutActivityUITests.kt

```diff
@@ -0,0 +1,347 @@
+        val mockPackageManager = mock<PackageManager> {
+            on { getPackageInfo(eq("protect.card_locker.contentprovider"), eq(0)) } doReturn PackageInfo().apply { versionName = "1.0.0" }
+        }
+        val mockResources = mock<Resources> {
+            on { getColor(any(), any()) } doReturn 0xFF000000.toInt() // Default color (black)
+        }
+
+
+
+        // Mock Context and PackageManager for AboutContent
+        mockContext = mock {
+            on { getString(eq(R.string.app_name)) } doReturn "Catima"
+            on { getString(eq(R.string.about_title_fmt), eq("Catima")) } doReturn "About Catima"
+            on { getString(eq(R.string.debug_version_fmt)) } doReturn "Version %s"
+            on { getString(eq(R.string.debug_version_fmt), eq("1.0.0")) } doReturn "Version 1.0.0"
+            on { getString(eq(R.string.app_copyright_fmt), eq(2025)) } doReturn "Copyright © 2025"
+            on { getString(eq(R.string.app_copyright_short)) } doReturn "Copyright 2023"
+            on { getString(eq(R.string.app_contributors), anyArray<String>()) } doReturn "Contributors: <br/>Test Contributor"
+            on { getString(eq(R.string.app_libraries), anyArray<String>()) } doReturn "Libraries: <br/>Test Library"
+            on { getString(eq(R.string.app_resources), anyArray<String>()) } doReturn "Resources: <br/>Test Asset"
+            on { getString(eq(R.string.app_copyright_old)) } doReturn "Old Copyright"
+            on { getString(eq(R.string.version_history)) } doReturn "Version history"
+            on { getString(eq(R.string.credits)) } doReturn "Credits"
+            on { getString(eq(R.string.help_translate_this_app)) } doReturn "Help translate this app"
+            on { getString(eq(R.string.license)) } doReturn "License"
+            on { getString(eq(R.string.source_repository)) } doReturn "Source repository"
+            on { getString(eq(R.string.privacy_policy)) } doReturn "Privacy policy"
+            on { getString(eq(R.string.donate)) } doReturn "Donate"
+            on { getString(eq(R.string.rate_this_app)) } doReturn "Rate this app"
+            on { getString(eq(R.string.report_error)) } doReturn "Report error"
+            on { getString(eq(R.string.translate_platform)) } doReturn "Weblate"
+            on { getString(eq(R.string.app_license)) } doReturn "MIT"
+            on { getString(eq(R.string.on_github)) } doReturn "GitHub"
+            on { getString(eq(R.string.and_data_usage)) } doReturn "and data usage"
+            on { getString(eq(R.string.ok)) } doReturn "OK"
+            on { getString(eq(R.string.view_online)) } doReturn "View online"
+            on { getPackageName() } doReturn "com.example.catima"
+            on { getPackageName() } doReturn "protect.card_locker.contentprovider"
+            on { packageManager } doReturn mockPackageManager
+            on {resources} doReturn mockResources
+            }
+
+        activity.content = MockAboutContent(mockContext)
```

Comment:
This causes the tests to fail on my device:

```
Cannot access 'var content: AboutContent': it is private in 'protect/card_locker/AboutActivity'
```

I think it should probably stay private though, I don't think we need the mocking (do we?)